commitizen/cli.py

```diff
@@ -131,6 +131,11 @@
                         ),
                         "action": "store_true",
                     },
+                    {
+                        "name": ["--annotated-tag", "-at"],
+                        "help": "create annotated tag instead of lightweight one",
```

Comment:
Maybe we can change the help message to `create an annotated tag (by default, commitizen creates lightweight tag)`